starfish/core/binary_mask/expand.py

```diff
@@ -4,14 +4,14 @@
 import xarray as xr
 
 from starfish.types import Axes
-from .util import _get_axes_names, AXES_ORDER
+from .util import _get_axes_names
 
 
 def fill_from_mask(
         mask: xr.DataArray,
         fill_value: int,
         result_array: np.ndarray,
-        axes_order: Sequence[Union[str, Axes]] = AXES_ORDER,
+        axes_order: Sequence[Union[str, Axes]] = (Axes.ZPLANE, Axes.Y, Axes.X),
```

Comment:
why not use your get_axes_order helper method here?

Reply:
That basically happens in #1628